chain/consensus/hierarchical/actors/sca/sca_state.go

```
@@ -104,9 +104,9 @@ func ConstructSCAState(store adt.Store, params *ConstructorParams) (*SCAState, e
 	if checkpointPeriod < minCheckpointPeriod {
 		checkpointPeriod = minCheckpointPeriod
 	}
-
+	sn, _ := address.SubnetIDFromString(params.NetworkName)
```

Why error is ignored here?